### PR TITLE
Support localization.

### PR DESCRIPTION
**Pull request by jianminLee**
Hi,I have localized support for this package, and it now supports localized languages ​​for permission searches. I have modified the README.md instructions.

### CODE REVIEW

File: resources/lang/zh-CN/navigation.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
```
Comment (paras-malhotra) [inline resources/lang/zh-CN/navigation.php]: Let's keep all lang files (except en) outside of the package. The user can always create these in their applications after they publish.

File: src/PermissionResourceTrait.php [hunk 2]
```diff
@@ -40,22 +41,63 @@ public function fields(Request $request)
...
+    /**
+     * Rewrite the applySearch method to apply translation field search
+     *
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  string  $search
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    protected static function applySearch($query, $search)
+    {
+        $trans_search = array_keys(preg_grep("/$search/",array_dot(__('laravel-nova-permission::permissions.display_names'))));
```
Comment (paras-malhotra) [inline src/PermissionResourceTrait.php]: This is also best left for the user's application.

**Comment by paras-malhotra** [comment]
@jianminLee thanks for the PR. I've requested a few changes. Once you make the changes, will merge.

File: src/PermissionResourceTrait.php [hunk 3]
```diff
@@ -40,22 +41,63 @@ public function fields(Request $request)
...
-			Text::make('Name', 'name')
+			Text::make(__('laravel-nova-permission::permissions.name'), 'name')
 				->rules(['required', 'string', 'max:255'])
 				->creationRules('unique:' . config('permission.table_names.permissions'))
 				->updateRules('unique:' . config('permission.table_names.permissions') . ',name,{{resourceId}}'),
 
-			Select::make('Guard Name', 'guard_name')
+            Text::make(__('laravel-nova-permission::permissions.display_name'),function (){
+                return __('laravel-nova-permission::permissions.display_names.'.$this->name);
+            })->canSee(function (){
```
Comment (paras-malhotra) [inline src/PermissionResourceTrait.php]: I think you'll need to correct indentation in all the changes you've made

**Review by paras-malhotra — CHANGES_REQUESTED** [review]

File: src/PermissionResourceTrait.php (hunk 2 above)
Comment (jianminLee) [reply src/PermissionResourceTrait.php]: I do not quite understand. If you remove the applySearch function, users will not be able to search for permissions through localized languages. Its existence will support "permission name" and "localization" searches.I think the default should be to use a localized "permission name", and the user only needs to decide whether to use it.

**Comment by paras-malhotra** [comment]
@jianminLee thanks for the PR. I have merged it but we are not supporting translated search in the package for now. If you need it, you can add your own Role Resource and Permission Resource classes. See https://github.com/insenseanalytics/laravel-nova-permission/commit/dbf7ef7e36fafcd5f5e4f25b3084b690824d78d9 for changes.

**Pull request merged**